run-scripts/run-kiosk-browser.sh

```diff
@@ -6,5 +6,5 @@ URL=${1:-http://localhost:3000}
 : "${VX_CONFIG_ROOT:="./config"}"
 
 
-kiosk-browser --add-file-perm o=http://localhost:3000,p=/media/**/*,rw --autoconfigure-print-config ./printing/printer-autoconfigure.json --url ${URL} --signify-secret-key ${VX_CONFIG_ROOT}/key.sec
+kiosk-browser --add-file-perm o=http://localhost:3000,p=/media/**/*,rw --add-file-perm o=http://localhost:3000,p=/var/log,ro --add-file-perm o=http://localhost:3000,p=/var/log/*,ro --autoconfigure-print-config ./printing/printer-autoconfigure.json --url ${URL} --signify-secret-key ${VX_CONFIG_ROOT}/key.sec --allow-devtools
```

Comment:
Time to break into multiple lines?

```suggestion
kiosk-browser \
  --add-file-perm o=http://localhost:3000,p=/media/**/*,rw \
  --add-file-perm o=http://localhost:3000,p=/var/log,ro \
  --add-file-perm o=http://localhost:3000,p=/var/log/*,ro \
  --autoconfigure-print-config ./printing/printer-autoconfigure.json \
  --url ${URL} \
  --signify-secret-key ${VX_CONFIG_ROOT}/key.sec
```

Also definitely do not want `--allow-devtools`.

Reply:
eek good catch, I just always have that stashed for using and totally forgot it was there

Reply:
`KIOSK_BROWSER_ALLOW_DEVTOOLS=true ./run.sh bsd` 😉